GameOfLife/Grid.cs

```diff
@@ -156,69 +151,64 @@ public void Update()
             age = cells[row, column].Age;
             count = CountNeighbors(row, column);
 
-            if (alive && count < 2)
-                return new Cell(row, column, 0, false);
-            
             if (alive && (count == 2 || count == 3))
             {
                 cells[row, column].Age++;
-                return new Cell(row, column, cells[row, column].Age, true);
+                return new Cell(row, column, age, true);
```

Comment:
here should be `cells[row, column].Age`, because on the line above you increase this value